Mexiletine/Mexiletine-Report.md

```diff
@@ -0,0 +1,366 @@
+
+![002_plotTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/002_plotTimeProfile.png)
+
+![003_plotPopulationTimeProfile.png](images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/003_plotPopulationTimeProfile.png)
+
+# 4 Conclusion
+The PBPK model developed for mexiletine was able to accurately predict the time-profiles following i.v. and p.o. dosing of mexiletine in EM and PM phenotypes. Observed variability was generally larger than the predicted in population simulations. Depending on study population, smoking status or variation in CYP-phenotypes may lead to additional variability which might be not included in the PK-Sim ontogeny factor.
+
+The predicted fraction excreted in urine was similar to the fraction reported in the label (9% vs 10%). The predicted bioavailability was 85%, compared to the observed as per label of 90%.
+
+# 5 References
+**Begg 1982** Begg, E., Chinwah, P., Webb, C., Day, R. & Wade, D. Enhanced metabolism of mexiletine after phenytoin administration. *British Journal of Clinical Pharmacology* **14**, 219–223 (1982).
+
+**Campbell 1978 ** Campbell NP, Kelly JG, Adgey AA, Shanks RG. Mexiletine in normal volunteers. *Br J Clin Pharmacol*. 1978;6(4):372-373.
+
+**DrugBank DB00379** (https://www.drugbank.ca/drugs/DB00379)
+
+**Joeres 1987** Joeres, R., Klinker, H., Heusler, H., Epping, J., Richter, E. (1987). Influence of mexiletine on caffeine elimination*. Pharmacology & therapeutics*, 33(1), 163-169.
+
+**Kuepfer 2016** Kuepfer L, Niederalt C, Wendl T, Schlender JF, Willmann S, Lippert J, Block M, Eissing T, Teutonico D. Applied Concepts in PBPK Modeling: How to Build a PBPK/PD Model.CPT Pharmacometrics Syst Pharmacol. 2016 Oct;5(10):516-531.
+
+**Kusumoto 1998 ** Kusumoto M, Ueno K, Tanaka K, et al. Lack of pharmacokinetic interaction between mexiletine and omeprazole. *Ann Pharmacother*. 1998;32(2):182-184. 
+
+**Labbé 2000** Labbé L, O’Hara G, Lefebvre M, et al. Pharmacokinetic and pharmacodynamic interaction be mexiletine and propafenone in human beings. *Clin Pharmacol Ther*. 2000;68(1):44-57.
+
+**Mexiletine, Drugs.com** Mexiletine, Drugs.com, Website drugs.com/pro/mexiletine.html
+
+**Pentikäinen 1984** Pentikainen P, Halinen M, Helin M. Pharmacokinetics of intravenous Mexiletine in patients with acute myocardial infarction. *J Cardiovasc Pharmacol*. 1984;6:1-6.
+
+**PK-Sim Ontogeny Database Version 7.3** (https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/blob/38cf71b384cfc25cfa0ce4d2f3addfd32757e13b/PK-Sim%20Ontogeny%20Database%20Version%207.3.pdf)
+
+**Pringle 1986** Pringle T, Fox J, McNeill J, et al. Dose independent pharmacokinetics of mexiletine in healthy volunteers. *Br J Clin Pharmacol*. 1986;21(3):319-321.
+
+**SmPC Namuscla** SmPC Namuscla 167 mg hard capsules, 2019, website medicines.org.uk/emc/product/9838/smpc
```

Comment:
Link not properly rendered. Use the full link (http://...

Reply:
Fixed